pkgs/applications/audio/ltc-tools/default.nix

```diff
@@ -0,0 +1,25 @@
+{stdenv, fetchFromGitHub, pkgconfig, libltc, libsndfile, jack2}:
+
+stdenv.mkDerivation rec {
+  name = "ltc-tools-${version}";
+  version = "0.6.4";
+
+  src = fetchFromGitHub {
+    owner = "x42";
+    repo = "ltc-tools";
+    rev = "v${version}";
+    sha256 = "1a7r99mwc7p5j5y453mrgph67wlznd674v4k2pfmlvc91s6lh44y";
+  };
+
+  buildInputs = [ pkgconfig libltc libsndfile jack2 ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/x42/ltc-tools";
+    description = "Tools to deal with linear-timecode (LTC)";
+    license = licenses.gpl2;
+    platforms = platforms.unix;
+    maintainers = with maintainers; [ ];
```

Comment:
Do you want to be the maintainer?

Reply:
@tg-x ^ Ideally you would also maintain the packages you contribute.

Otherwise everything seems in order here. (I guess from previous packages contributions you won't add yourself.) I guess in a couple more days without a response it can be merged.

Reply:
@nlewo @samueldr yes i was going to add myself, should be fixed now.